src/components/Shipping.js

```diff
@@ -0,0 +1,51 @@
+import { __ } from "@wordpress/i18n";
+import { Button, Badge } from "@yoast/ui-library";
+import { ReactComponent as Shippo } from "../icons/shippo.svg";
+
+const Shipping = ({ integrationStatus, onConnect }) => {
+  const isSetupComplete = integrationStatus?.complete;
+  const environment = integrationStatus?.details?.environment;
+
+  return (
+    <div className="yst-h-[174px] yst-border yst-h-174px yst-p-6">
+      <div className="yst-flex yst-justify-between yst-mb-8">
+        <Shippo />
+        <Button
```

Comment:
I think readability wise, it might be simpler to render two different `Button` based on `isSetupComplete`